backend/src/test/java/com/yohoard/notes/FirefoxImporterTest.java

```diff
@@ -0,0 +1,59 @@
+package com.yohoard.notes;
+
+import com.yohoard.notes.importer.firefox.FirefoxImporter;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.testcontainers.service.connection.ServiceConnection;
+import org.springframework.data.domain.Example;
+import org.testcontainers.containers.MongoDBContainer;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+import java.io.IOException;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@SpringBootTest
+@Testcontainers
```

Comment:
Zastanawia mnie jak dobrze znamy i umiemy się posługiwać test containerami by brać je do projektu :P Myślisz, że to gdzieś nie zaskoczy z nienacka w nieprzyjemny sposób? Ogólnie wydaje się fajne i koncepcje ma dobrą

Reply:
Myślę, że warto spróbować, duplikując te parę linijek konfiguracji testcontainers w innych testach lub dając do nadrzędnej klasy abstrakcyjnej powinno wystarczyć żeby testy działały poprawnie, tak jak w tym przypadku